src/App.tsx

```diff
@@ -1,15 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-export const App: React.FC = () => {
+export const App: React.FC<Movie> = () => {
+  const [query, setQuery] = useState('');
+  const filterMovies = (event:
+  { currentTarget:
+  { value: React.SetStateAction<string>; }; }) => setQuery(
+    event.currentTarget.value,
+  );
+
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    return Object.values(movie)
+      .join('')
+      .toLowerCase()
+      .includes(query
+        .toLowerCase());
```

Comment:
According to conditions to need
 - check if `movie.title` or `movie.description` contains `query`;
Not for all fields - only for title and description. 
Your decision will filter '1375' , but it isn't at title or description.
![image](https://user-images.githubusercontent.com/105741133/191076679-a75e9129-1aef-4f09-9a61-6a5f882b1f96.png)